src/kata3.numberToReversedDigits.js

```diff
@@ -1,5 +1,12 @@
 const numberToReversedDigits = (number) => {
 
+const tostring = number.toString();
+const split = tostring.split('');
+let map = split.map(function(result) {
```

Comment:
`map` should be a constant variable here as it isn't reassigned to. I'd also use an arrow function with an implicit return:

```js
const map = split.map(result => parseInt(result));
```